### PR TITLE
deps update

### PR DESCRIPTION
**Pull request by floridemai**

### CODE REVIEW

File: ios/COVIDGreen.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -515,7 +515,7 @@
...
...$_JSON_OUTPUT_BASE64\")\n  echo \"warning:   A firebase.json file was not found, whilst this file is optional it is recommended to include it to configure firebase services in React Native Firebase.\"\nfi;\n\necho \"info: 2) Injecting Info.plist entries: \"\n\n# Log out the keys we're adding\nfor i in \"${!_PLIST_ENTRY_KEYS[@]}\"; do\n  echo \"    ->  $i) ${_PLIST_ENTRY_KEYS[$i]}\" \"${_PLIST_ENTRY_TYPES[$i]}\" \"${_PLIST_ENTRY_VALUES[$i]}\"\ndone\n\nfor plist in \"${_TARGET_PLIST}\" \"${_DSYM_PLIST}\" ; do\n  if [[ -f \"${plist}\" ]]; then\n\n    # paths with spaces break the call to setPlistValue. temporarily modify\n    # the shell internal field separator variable (IFS), which normally\n    # includes spaces, to consist only of line breaks\n    oldifs=$IFS\n    IFS=\"\n\"\n\n    for i in \"${!_PLIST_ENTRY_KEYS[@]}\"; do\n      setPlistValue \"${_PLIST_ENTRY_KEYS[$i]}\" \"${_PLIST_ENTRY_TYPES[$i]}\" \"${_PLIST_ENTRY_VALUES[$i]}\" \"${plist}\"\n    done\n\n    # restore the original internal field separator value\n    IFS=$oldifs\n  else\n    echo \"warning:   A Info.plist build output file was not found (${plist})\"\n  fi\ndone\n\necho \"info: <- RNFB build script finished\"\n";
```
Comment (AlanSl) [inline ios/COVIDGreen.xcodeproj/project.pbxproj:518]: Of all the changes for GitHub to choose not to highlight individual characters...

`usefl` -> `useful` and dropped one `\n` at the end right?

File: ios/Podfile.lock [hunk 2]
```diff
@@ -254,13 +254,13 @@ PODS:
     - React-cxxreact (= 0.63.2)
     - React-jsi (= 0.63.2)
   - React-jsinspector (0.63.2)
-  - react-native-exposure-notification-service (1.1.28):
+  - react-native-exposure-notification-service (1.1.30):
```
Comment (AlanSl) [inline ios/Podfile.lock:257]: Do we need to bump this in the package-json too?

**Review by AlanSl — COMMENTED** [review]
Bit nervy about doing this last minute... trusting you know what you're doing for each change and have tested it all thoroughly. Couple of questions

File: ios/Podfile.lock (hunk 2 above)
Comment (AlanSl) [reply ios/Podfile.lock:257]:
```
   "react-native-exposure-notification-service": "^1.1.28",
```

File: ios/Podfile.lock [hunk 3]
```diff
@@ -336,23 +336,23 @@ PODS:
...
     - React
   - RNDeviceInfo (5.6.5):
     - React
-  - RNFBApp (8.4.1):
+  - RNFBApp (8.4.3):
     - Firebase/CoreOnly (~> 6.30.0)
-    - React
-  - RNFBCrashlytics (8.4.2):
+    - React-Core
+  - RNFBCrashlytics (8.4.5):
```
Comment (AlanSl) [inline ios/Podfile.lock:346]: Wasn't this removed? Is there a reason the pods are still included?

File: ios/COVIDGreen.xcodeproj/project.pbxproj (hunk 1 above)
Comment (floridemai) [reply ios/COVIDGreen.xcodeproj/project.pbxproj:518]: That and a space. But it was done automatically.

File: ios/Podfile.lock (hunk 2 above)
Comment (floridemai) [reply ios/Podfile.lock:257]: No need i think, as yarn lock is up to date and so is the Podfile.lock (actually this is the main reason I did this update).

File: ios/Podfile.lock (hunk 3 above)
Comment (floridemai) [reply ios/Podfile.lock:346]: I wasn't aware I just saw a flag on false, I won't touch it further

**Review by AlanSl — APPROVED** [review]
Trusting you've tested this fully

**Comment by AlanSl** [comment]
@floridemai Would now be a good time to apply this (or close it if it's obsolete)?

**Comment by floridemai** [comment]
@AlanSl Haha, I've closed it without seeing your comment :)

**Pull request closed without merging**